app/src/lib/menu-update.ts

```diff
@@ -164,12 +166,29 @@ function getMenuState(state: IAppState): Map<MenuIDs, IMenuItemState> {
     menuStateBuilder.disable('pull')
   }
 
+  const welcomeScopedIds: ReadonlyArray<MenuIDs> = [
```

Comment:
We should probably include the about and preferences menu items here as well since we're not showing any dialogs while in the welcome flow